azure-pipelines.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         -e NRN_RELEASE_UPLOAD \
         -e NEURON_WHEEL_VERSION \
         -e NRN_BUILD_FOR_UPLOAD=1 \
-        'neuronsimulator/neuron_wheel' \
+        'neuronsimulator/neuron_wheel_ml2014' \
```

Comment:
This will be reverted before merging and `neuron_wheel_ml2014` deleted. If `manylinux1` image will be needed in the future, then one can specify `neuronsimulator/neuron_wheel:manylinux1`